lang/csharp/src/apache/main/Avro.main.csproj

```diff
@@ -49,7 +49,21 @@
 
   <!-- See lang/csharp/README.md for tool and library dependency update strategy -->
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="4.0.1">
```

Comment:
Code styling related package referncing is handled by common.props, to make sure that every projects inherit the same codestyling

Reply:
Chage was deleted